api/v1/views/states.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+""" index file """
+
+from api.v1.views import app_views
+from flask import jsonify
+from models.state import State
+
+@app_views.route('/states')
+def states():
+    """ status function """
+    return jsonify(storage.all("State"))
+
+@app_views.route('/states/<string:state_id>', methods=['GET'])
+def get_state():
+    """"""
+    return jsonify(storage.get("State", state_id))
```

Comment:
It seems that it is for the task 7. I have not read it yet, so I don't understand as well the thing.